src/embed.ts

```diff
@@ -17,6 +17,12 @@ const embeddingModels = ['textembedding-gecko'] as const;
 export interface EmbedOptions {
   model?: EmbedModel;
```

Comment:
I'll fix this in a follow up, don't want this to be optional tbh

Reply:
I can change it now.